index.html

```diff
@@ -2825,7 +2825,10 @@ <h4>DagCBOR</h4>
           </ul>
 
           <pre class="example nohighlight" title="DID Document as DagCBOR same as the previous example, but serialized to JSON for easy readability">
-{ "@context": "https://www.w3.org/ns/did/v1",
+{ "@context": [
+    "https://www.w3.org/ns/did/v1",
+    "https://w3id.org/security/v2"
+  ],
```

Comment:
Hrm, we shouldn't pull in https://w3id.org/security/v2 -- the terms that are missing from the vocabulary should be defined in did/v1 -- do you know what was missing @gjgd?

Reply:
`proof`

Reply:
<img width="896" alt="Screenshot 2020-10-08 at 15 57 20" src="https://user-images.githubusercontent.com/7913565/95468678-0d166480-097f-11eb-9d25-cb5387878c76.png">

From [json ld checker](https://www.jsonld-checker.com/?json=%257B%250A%2520%2520%2522%2540context%2522%253A%2520%2522https%253A%252F%252Fwww.w3.org%252Fns%252Fdid%252Fv1%2522%252C%250A%2520%2520%2522authentication%2522%253A%2520%255B%250A%2520%2520%2520%2520%2522did%253Aexample%253A12D3KooWMHdrzcwpjbdrZs5GGqERAvcgqX3b5dpuPtPa9ot69yew%253Bkey-id%253Dbafyreicubtx5wqo3nosc4cazrkctfhwd6rewezgpwoe4swirls4ebdhs2i%2522%250A%2520%2520%255D%252C%250A%2520%2520%2522created%2522%253A%2520%25222018-12-01T03%253A00%253A00Z%2522%252C%250A%2520%2520%2522id%2522%253A%2520%2522did%253Aexample%253A12D3KooWMHdrzcwpjbdrZs5GGqERAvcgqX3b5dpuPtPa9ot69yew%2522%252C%250A%2520%2520%2522proof%2522%253A%2520%257B%250A%2520%2520%2520%2520%2522created%2522%253A%2520%25222020-05-01T03%253A00%253A02Z%2522%252C%250A%2520%2520%2520%2520%2522creator%2522%253A%2520%2522did%253Aexample%253A12D3KooWMHdrzcwpjbdrZs5GGqERAvcgqX3b5dpuPtPa9ot69yew%253B%2520example%253Akey%253Did%253Dbafyreicubtx5wqo3nosc4cazrkctfhwd6rewezgpwoe4swirls4ebdhs2i%2522%252C%250A%2520%2520%2520%2520%2522signatureValue%2522%253A%2520%2522o9r6LxgoGN8FoaeeUA6EdDcv12GvDzFEmCgjWzvpur2YSQyA8W2r0SSWUK%252BnH5tMqzaFLun6wwZ1Eot37amGDg%253D%253D%2522%252C%250A%2520%2520%2520%2520%2522type%2522%253A%2520%2522ed25519Signature2018%2522%250A%2520%2520%257D%252C%250A%2520%2520%2522verificationMethod%2522%253A%2520%255B%250A%2520%2520%2520%2520%257B%250A%2520%2520%2520%2520%2520%2520%2522curve%2522%253A%2520%2522ed25519%2522%252C%250A%2520%2520%2520%2520%2520%2520%2522expires%2522%253A%2520%25222019-12-01T03%253A00%253A00Z%2522%252C%250A%2520%2520%2520%2520%2520%2520%2522id%2522%253A%2520%2522bafyreicubtx5wqo3nosc4cazrkctfhwd6rewezgpwoe4swirls4ebdhs2i%253Bexample%253Akey%253Didbafyreicubtx5wqo3nosc4cazrkctfhwd6rewezgpwoe4swirls4ebdhs2i%2522%252C%250A%2520%2520%2520%2520%2520%2520%2522publicKeyBase64%2522%253A%2520%2522qmz7tpLNKKKdl7cD7PbejDiBVp7ONpmZbfmc7cEK9mg%253D%2522%252C%250A%2520%2520%2520%2520%2520%2520%2522type%2522%253A%2520%2522EdDsaPublicKey%2522%250A%2520%2520%2520%2520%257D%250A%2520%2520%255D%252C%250A%2520%2520%2522updated%2522%253A%2520%25222020-05-01T03%253A00%253A00Z%2522%250A%257D&tab=0)

Reply:
Doh! I think this snuck in there with the DAG CBOR PR -- agreed, @gjgd, the right thing to do here is remove `proof`. /cc @jonnycrunch -- please make sure you don't include `proof` in future examples. It was removed from the spec a while ago.

Reply:
removal of proof came as a surprise to me.  [Recent PR](https://github.com/w3c/did-core/pull/420) with updated CBOR section it has been removed.

Reply:
Will do, though I think we also need to remove `verificationMethod`. I'm not an expert in DagCBOR, would the resulting document (without proof and verificationMethod) still illustrate [the corresponding section](https://w3c.github.io/did-core/#dagcbor) @msporny ?

Reply:
See https://github.com/w3c/did-spec-registries/blob/master/contexts/did-v1.jsonld, IMO, the example should match terms that are defined in that file.